docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -261,9 +263,13 @@ The prop accepts an object of type [GridSlotsComponent](/api/data-grid/#slots) .
 
 ### Toolbar
 
-You can provide your own toolbar by passing it to the `Toolbar` component.
+To enable the toolbar you need to add the `Toolbar: GridHeader` to the `components` grid prop.
 This demo showcases how this can be achieve.
 
+{{"demo": "pages/components/data-grid/rendering/ToolbarGrid.js", "bg": "inline"}}
+
+Alternatively you can compose your own toolbar.
```

Comment:
I think that we should avoid the need to open the source:

````suggestion
Alternatively, you can compose your own toolbar.

```jsx
function CustomToolbar() {
  return (
    <GridToolbarContainer>
      <ColumnsToolbarButton />
      <FilterToolbarButton />
    </GridToolbarContainer>
  );
}
```
````